passkeeper/__init__.py

```diff
@@ -77,6 +77,20 @@ def cleanup_ini(self):
                 LOG.info('Clean file %s' % fname)
                 run_cmd('shred --remove %s' % file_path)
 
+    def cleanup_history(self, directory):
+        "Shred all .git files"
+        for root, dirs, files in os.walk(directory, topdown=False):
+            for fname in files:
+                filepath = os.path.join(root, fname)
+                LOG.info('Clean file %s' % fname)
+                run_cmd('shred -f --remove %s' % filepath)
+            for dname in dirs:
+                dpath = os.path.join(root, dname)
+                run_cmd('rmdir %s' % dpath)
```

Comment:
I think we should avoid to call system command for each action.

Prefer use python command to remove file or directory .It allow you to have exception and detailed error.

 `shred` is run with a  `run_cmd` because I have not found effective way to do this in python.